src/guide/jbs-jdk-bug-system.md

```diff
@@ -122,11 +122,11 @@ Add a comment when adding a (Rel)[-wnf]{.jbs-label} label so that it's clear for
 
 ![Guidelines for setting [Affects Version/s]{.jbs-field}](affects_versions.svg)
 
-1. Issue relevant to JDK 8 and all future releases.
-2. No need to add additional releases as they are implied.
+1. Affects Version contains 8 only, which means this issue is applicable to 8 and all future releases (until it's fixed).
+2. Affects Version contains 8, 8u40, 9, and 12, yet 8 already implies that the later releases are affected.
```

Comment:
I used strike-through to indicate 8u40, 9, and 12 aren't need, which could also be confusing. Therefore, it makes to explicitly state that 8u40, 9, and 12 should not be added to the Affects Version field.